Use 'import from git'-functionality

Fuzzbender opened the pull request:
There is a new functionality available in 'cyberismo'. You can import modules directly from git instead of first cloning the repositories and then importing from filesystem. As this removes one whole step in tutorial use it in the docs example.

Fuzzbender commented:
To make git-imports work, the environment variables (user and token needs to be set).
Don't approve this right now - we need to figure out how to provide TOKEN to user (it should be fine to import public ones without the token).

henryhaverinen commented:
> To make git-imports work, the environment variables (user and token needs to be set). Don't approve this right now - we need to figure out how to provide TOKEN to user (it should be fine to import public ones without the token).

Sorry about approving this

Fuzzbender commented:
> > To make git-imports work, the environment variables (user and token needs to be set). Don't approve this right now - we need to figure out how to provide TOKEN to user (it should be fine to import public ones without the token).
> 
> Sorry about approving this

It is fine, this can wait until we have all of the infra set.

Fuzzbender commented:
Once https://github.com/CyberismoCom/cyberismo/pull/612 has been merged; we can continue with this PR.

Fuzzbender commented:
Updated the PR. 
Removed unnecessary changes to `index.json` (timestamp update).  
Changed all referred locations where `import module` is being used.
